### PR TITLE
Add blog post for 0.2 release.

### PR DESCRIPTION
**Pull request by robertnishihara**
This post briefly mentions
- plasma performance improvements
- the web UI
- RLlib
- actor fault tolerance

Some of these things will need follow up blog posts with more elaboration.

**Questions:**
- The timeline screenshot is a bit large (34KB). Do you think that's a problem? It used to be 122KB before I compressed it with http://compresspng.com/.

@atumanov @stephanie-wang @ericl @richardliaw @pcmoritz @alanamarzoev please take a look.

### CODE REVIEW

File: site/_posts/2017-09-27-ray-0.2-release.markdown [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+title: "Ray: 0.2 Release"
+excerpt: "This post announces the release of Ray 0.2."
+date: 2017-09-27 14:00:00
+---
+
+We are pleased to announce the Ray 0.2 release. This release includes the
+following:
+- substantial [performance improvements to the Plasma object store][1]
+- an initial [Jupyter notebook based web UI][2]
+- the start of a [scalable reinforcement learning library][3]
```
Comment (richardliaw) [inline site/_posts/2017-09-27-ray-0.2-release.markdown]: "start" sounds odd, but I'm not sure what fits here

File: site/_posts/2017-09-27-ray-0.2-release.markdown [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+humanoid robot in the Mujoco physics engine.
+
+Running this on AWS with a cluster of fifteen m4.16xlarge instances and one
+p2.16xlarge instance, we achieve a reward of over 6000 in around 35 minutes. The
+rollouts are parallelized over 512 physical cores and the policy optimization is
+parallelized over 6 GPUs. Relevant hyperparameters for this experiment are
+[here][8].
+
+This RL library is under development, and we are looking for contributions
+including implementations of more algorithms.
```
Comment (richardliaw) [inline site/_posts/2017-09-27-ray-0.2-release.markdown]: Maybe also mention another upcoming post about RLLib specifically?

File: site/_posts/2017-09-27-ray-0.2-release.markdown [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+python ray/python/ray/rllib/train.py --alg=PPO \
+                                     --env=Humanoid-v1 \
+                                     --redis-address=<head-node-ip>:6379
+```
+
+This uses [proximal policy optimization][11] to train a policy to control a walking
+humanoid robot in the Mujoco physics engine.
+
+Running this on AWS with a cluster of fifteen m4.16xlarge instances and one
+p2.16xlarge instance, we achieve a reward of over 6000 in around 35 minutes. The
```
Comment (richardliaw) [inline site/_posts/2017-09-27-ray-0.2-release.markdown]: update this with new results?

File: site/_posts/2017-09-27-ray-0.2-release.markdown [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+time.sleep(1)
+
+x = 1
+for _ in range(3):
+    ys = [f.remote(x) for _ in range(8)]
+    x = g.remote(*ys)
+```
+
+## RLlib
```
Comment (richardliaw) [inline site/_posts/2017-09-27-ray-0.2-release.markdown]: RLLib or RLlib?

File: site/_posts/2017-09-27-ray-0.2-release.markdown (hunk 3 above)
Comment (robertnishihara) [reply site/_posts/2017-09-27-ray-0.2-release.markdown]: I think these are the new results, aren't they?

Comment (richardliaw) [reply site/_posts/2017-09-27-ray-0.2-release.markdown]: oh right, my bad

**Pull request merged**